applications/app/services/TagFrontPicker.scala

```diff
@@ -60,7 +60,7 @@ object TagFrontPicker extends GuLogging {
     val properties =
       Map(
         "participatingInTest" -> participatingInTest.toString,
-//        "testPercentage" -> DCRTagFronts.participationGroup.percentage,
+        "testPercentage" -> DCRTagFronts.participationGroup.percentage,
```

Comment:
This reminds me, do we need a dashboard for tag pages?

Reply:
wouldn't that be absolutely lovely! yes I can make one for sure, should be a simple copy of the fronts one 👍

Reply:
here we are https://metrics.gutools.co.uk/d/c1b51de3-ee8e-430d-ac5e-2f028338a26f/tag-page-migration?orgId=1